ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.20__amd64-nightly-4.20-upgrade-from-stable-4.19.yaml

```diff
@@ -122,6 +122,7 @@ tests:
       EXTRACT_MANIFEST_INCLUDED: "true"
       FIPS_ENABLED: "true"
       TEST_FILTERS: ~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+      UPGRADE_CCO_MANUAL_MODE: oidc
```

Comment:
There is only one profile `aws-ipi-disc-priv-sts-ep-fips-f28` enabled the var. But I see there are more profiles in openshift-openshift-tests-private-release-4.20__amd64-nightly-4.20-upgrade-from-stable-4.19.yaml

```
$ grep -r "as: " openshift-openshift-tests-private-release-4.20__amd64-nightly-4.20-upgrade-from-stable-4.19.yaml|grep aws.*sts
- as: aws-ipi-disc-priv-sts-ep-fips-f28
- as: aws-ipi-localzone-byo-subnet-sts-mini-perm-f28
- as: aws-ipi-localzone-sts-fips-mini-perm-f28
- as: aws-ipi-private-shared-vpc-phz-sts-canary-f28
- as: aws-ipi-shared-vpc-phz-sts-fips-openldap-mini-perm-f28
- as: aws-ipi-wavelength-byo-subnet-sts-mini-perm-f28
- as: aws-ipi-wavelength-sts-fips-mini-perm-f28
- as: aws-usgov-ipi-private-sts-f28
```